hawc/apps/assessment/templates/assessment/assessmentvalue_detail.html

```diff
@@ -65,6 +65,15 @@ <h3>Assessment Value for {{object.assessment}}</h3>
       {% endfor %}
     </tbody>
   </table>
+  {% if pod_message %}
+    <div class="usa-alert usa-alert--info usa-alert--slim">
+      <div class="usa-alert__body">
+        <p class="usa-alert__text">
+          {{pod_message}}
```

Comment:
write the message here; and make `pod_message` the values you need to add the text